students/11th/daseulsong/main.html

```diff
@@ -0,0 +1,292 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width" />
+    <title>main-page</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link href="style/common.css" rel="stylesheet" type="text/css" />
+    <link href="style/main.css" rel="stylesheet" type="text/css" />
+  </head>
+  <script
+    src="https://kit.fontawesome.com/f2f9d53b4c.js"
+    crossorigin="anonymous"
+  ></script>
+  <script src="js/main.js" defer></script>
+  <body>
+    <header>
+      <div class="header__container">
+        <ul class="logo">
+          <li><i class="fab fa-instagram"></i></li>
+          <li><p>|</p></li>
+          <li><img src="img/logo_text.png" /></li>
```

Comment:
```suggestion
          <li><img alt="logo" src="img/logo_text.png" /></li>
```
img 태그 사용할 때 alt 속성잊지 말아주세요

Reply:
옙 🙂 alt 속성 부여했습니다.